### ISSUE TITLE
Show breadcrumbs to current topic

### ISSUE THREAD

**Issue by littledan** [bterlson/tcq#18]
During a reply, show the description of the parent topic in the UI at the same time. Also, consider allowing returning to the parent topic after a reply is done. This way, speakers won't feel like they are "giving up their slot" by clicking done and working down through the replies.

**Comment by bterlson** [comment]
Good idea. Related to #5 (adding current agenda item to the UI as well).
